sdk/rust/src/main.rs

```diff
@@ -1,28 +1,33 @@
 use tokio;
-//use serde_json::{json, Map, Value};
-//use serde::{Serialize, Deserialize};
-//use reqwest::{Client, ClientBuilder, Result, Error};
 
 use ehsm_client::{client::EHSMClient, api::KMS};
+use base64::{encode};
 
-const APP_ID: &str = "011249a4-2f17-4380-8922-a90ff5dd3976";
-const API_KEY: &str = "Zgeiig9SFvNy6vWRpSQMSFyrWUfEYnnG";
-const BASE_URL: &str = "https://10.112.240.169:9002/ehsm?Action=";
+const APP_ID: &str = "a76b9886-d1e6-4a77-98d8-4f2c630f1f5e";
+const API_KEY: &str = "FWMs1EypjsBcUFf07qA0N8z3wAcTrWA9";
+const BASE_URL: &str = "https://127.0.0.1:9002/ehsm?Action=";
+const PLAINT_TEXT: &str = "Intel test";
 
```

Comment:
need to define in env variable. you can check the implementation of go, and keep the same way.

Reply:
ok